modules/administration-guide/partials/assembly_authenticating-users-on-3rd-party-services.adoc

```diff
@@ -0,0 +1,16 @@
+// securing-che
+
+:parent-context-of-authorizing-users: {context}
+
+[id="authenticating-users-on-3rd-party-services_{context}"]
+= Authenticating users on 3-rd party services
+
+:context: authenticating-users-on-3rd-party-services
+
+This topic covers authentication of users on 3-rd party services such as Bitbucket, GitLab etc
```

Comment:
That's not true, it does not cover authentication on Gitlab, does it?

Reply:
removed mention for now